src/components/Search/Filters/Filter.module.scss

```diff
@@ -0,0 +1,15 @@
+@use "src/styles/breakpoints";
+
+.dropdownLabel {
+  font-weight: var(--font-weight-bold);
+  font-size: var(--font-size-small);
+}
+
+.comboboxItems {
+  div[role="dialog"] {
+    width: 82%;
```

Comment:
not sure if there were hardcoded from before but probably shouldn't be hardcoded.